tests/mockers.py

```diff
@@ -78,6 +78,12 @@ def commit(self, message: str = "Some commit message.") -> "GitRepositorySandbox
         self.execute(f'git commit -m "{message}"')
         return self
 
+    def add_file_with_content_and_commit(self, file_name: str = 'file_name.txt', file_content: str = 'Some file content', message: str = "Some commit message.") -> "GitRepositorySandbox":
+        self.execute(f"echo {file_content} > {file_name}")
```

Comment:
I'd wrap file content in apostrophes to avoid surprises
```suggestion
        self.execute(f"echo '{file_content}' > {file_name}")
```
Or best, just use some Python API for writing the file, rather than shell

Reply:
Ok, i wanted to keep the style of only using shell in the function but i guess it's not necessary

Reply:
Yeah, shell invocations are still much more brittle than pure Python